contracts/eosio.system/include/eosio.system/eosio.system.hpp

```diff
@@ -95,19 +95,24 @@ namespace eosiosystem {
    using blockchain_parameters_t = eosio::blockchain_parameters;
 #endif
 
-  /**
-   * The `eosio.system` smart contract is provided by `block.one` as a sample system contract, and it defines the structures and actions needed for blockchain's core functionality.
-   *
-   * Just like in the `eosio.bios` sample contract implementation, there are a few actions which are not implemented at the contract level (`newaccount`, `updateauth`, `deleteauth`, `linkauth`, `unlinkauth`, `canceldelay`, `onerror`, `setabi`, `setcode`), they are just declared in the contract so they will show in the contract's ABI and users will be able to push those actions to the chain via the account holding the `eosio.system` contract, but the implementation is at the EOSIO core level. They are referred to as EOSIO native actions.
-   *
-   * - Users can stake tokens for CPU and Network bandwidth, and then vote for producers or
-   *    delegate their vote to a proxy.
-   * - Producers register in order to be voted for, and can claim per-block and per-vote rewards.
-   * - Users can buy and sell RAM at a market-determined price.
-   * - Users can bid on premium names.
-   * - A resource exchange system (REX) allows token holders to lend their tokens,
-   *    and users to rent CPU and Network resources in return for a market-determined fee.
-   */
+   /**
+    * The `eosio.system` smart contract is provided by `block.one` as a sample system contract, and it defines the
+    * structures and actions needed for blockchain's core functionality.
+    *
+    * Just like in the `eosio.bios` sample contract implementation, there are a few actions which are not implemented at
+    * the contract level (`newaccount`, `updateauth`, `deleteauth`, `linkauth`, `unlinkauth`, `canceldelay`, `onerror`,
+    * `setabi`, `setcode`), they are just declared in the contract so they will show in the contract's ABI and users
+    * will be able to push those actions to the chain via the account holding the `eosio.system` contract, but the
+    * implementation is at the EOSIO core level. They are referred to as EOSIO native actions.
```

Comment:
You only reformatted this comment, but does seem like `EOSIO` is not what we want to call the native actions. I think just remove `EOSIO`.
`implementation is at the core level. They are referred to as host native actions.`

Reply:
#419b696